### ISSUE TITLE
Can this be extended to TouchEvents?

### ISSUE THREAD

**Issue by HarikalarKutusu** [yomotsu/hold-event#3]
Thank you for this also, it saved a lot of work for me in my Virtual Exhibition. This works perfectly with keyboard and movement icons. Can this be extended to touchevents?

I read about this a bit and I can see the problem. Even nipplejs lacks the "hold" support. Or do you know a working method/npn/git for this purpose?

**Comment by yomotsu** [comment]
It's possible. However, a little bit complicated since touchEvent takes multiple fingers.
I can work on this when I have time, but if you need it asap, PR is also welcome!
https://github.com/yomotsu/hold-event/blob/master/src/ElementHold.ts#L14-L18

Also, I have another lib that allows touch events.
https://github.com/yomotsu/screen-gamepad

**Comment by HarikalarKutusu** [comment]
Yes, I saw the complication. What I was thinking was something like mouse emulation we were using on single-touch computer screens ten years ago. If connected to Element, one can use an icon-button on both regular desktops and mobile devices.

Unfortunately, I'm not proficient enough to modify your library, but I'll be happy to be a beta tester on multiple devices.

My current implementation just connects the "touchstart" event with fixed delta (selected 200). As mentioned elsewhere on the Internet, I could set a flag and continue the movement until it is unset by a "touchend" event, but animations in camera-controls would make it jumpy (I think).
